src/dbus_api/pool.rs

```diff
@@ -30,14 +30,8 @@ use super::blockdev::create_dbus_blockdev;
 use super::filesystem::create_dbus_filesystem;
 use super::types::{DbusContext, DbusErrorEnum, OPContext, TData};
 
-use super::util::STRATIS_BASE_PATH;
-use super::util::STRATIS_BASE_SERVICE;
-use super::util::code_to_message_items;
-use super::util::engine_to_dbus_err;
-use super::util::get_next_arg;
-use super::util::get_uuid;
-use super::util::ok_message_items;
-
+use super::util::{code_to_message_items, default_object_path, engine_to_dbus_err, get_next_arg,
+                  get_uuid, ok_message_items, STRATIS_BASE_PATH, STRATIS_BASE_SERVICE};
```

Comment:
Did you do this manually or did rustfmt do this magically? Just curious.

Reply:
I combined the lines together manually.